web_help/static/src/trip.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="web_help.TripStep" owl="1">
+        <div>
+            <div class="pb-4">
+                <t t-raw="content" />
+            </div>
+
+            <button
+                class="btn btn-lg btn-outline-light web_help_cb_button mr8"
+            >Got it</button>
```

Comment:
```suggestion
            >Next</button>
```

Nitpicking + Not blocking : I find that "Got it" can mean "stop the trip", while "Next" is more explicit.

Reply:
I've changed it so it's easy to change the button text or the whole template.